### PR TITLE
Update publish-build-assets.yml

### PR DESCRIPTION
**Pull request by JohnTortugo**

### CODE REVIEW

File: eng/common/templates/job/publish-build-assets.yml [hunk 1]
```diff
@@ -57,7 +57,6 @@ jobs:
           /p:MaestroApiEndpoint=https://maestro-prod.westus2.cloudapp.azure.com
           /p:PublishUsingPipelines=${{ parameters.publishUsingPipelines }}
           /p:Configuration=$(_BuildConfig)
```
Comment (jcagme) [inline eng/common/templates/job/publish-build-assets.yml]: What's the default verbosity?

Comment (jcagme) [inline eng/common/templates/job/publish-build-assets.yml]: What's the default verbosity?

Comment (jcagme) [inline eng/common/templates/job/publish-build-assets.yml]: What's the default verbosity?

Comment (JohnTortugo) [reply eng/common/templates/job/publish-build-assets.yml]: 'normal'

**Pull request merged**